src/pQueue.js

```diff
@@ -0,0 +1,39 @@
+class Node {
+  constructor(key, value) {
+    this.key = value
+  }
+}
+
+export default class PQueue {
+  constructor( things, stuff) {
+  this.front = new Node(things, stuff)
+  }
```

Comment:
It wouldn't be an honest review if I let the unintuitive variable names slide, but I know this project is in progress so just planting a seed to change them later once everything makes more sense :)